packages/framework/src/Framework/Features/Publications/Models/PublicationFieldType.php

```diff
@@ -91,26 +93,26 @@ public function getValidationRules(bool $reload = true): Collection
             case 'integer':
             case 'string':
             case 'text':
-                if ($doBetween) {
+                if ($useRange) {
                     $fieldRules->add("between:$this->min,$this->max");
                 }
                 break;
             case 'image':
                 $mediaFiles = PublicationService::getMediaForPubType($this->publicationType, $reload);
                 $valueList = $mediaFiles->implode(',');
-                $fieldRules->add("in:$valueList");
+                $fieldRules->add("in:$valueList"); // FIXME What if the list is empty?
+                                                   // FIXME: Now the items look like 'in:_media/foo/bar.jpg', but do we really need the directory information?
+                                                   //   Wouldn't it suffice with just 'in:bar.jpg' since we already know what directory it is in?
+                                                   //   We could then easily qualify it within the template and/or via a helper method.
                 break;
             case 'tag':
-                $tagValues = PublicationService::getValuesForTagName($this->tagGroup, $reload);
+                $tagValues = PublicationService::getValuesForTagName($this->tagGroup, $reload) ?? collect([]);
                 $valueList = $tagValues->implode(',');
                 $fieldRules->add("in:$valueList");
                 break;
             case 'url':
+                // FIXME Shouldn't we add a 'url' rule here?
                 break;
-            default:
```

Comment:
Default case removed as it's not possible to reach since the backed enum is evaluated in the constructor.